lib/Config/Initializer.php

```diff
@@ -346,6 +349,13 @@ public function handleFailedInitialization($exception) {
     return WPNotice::displayError($exception);
   }
 
+  private function handleRunningInitialization(InvalidStateException $exception) {
```

Comment:
Hey, this is probably not a great name, here should probably be `handleRunningMigration`, right?

Reply:
Thanks, this name will be better.